src/libical/icaltime.c

```diff
@@ -656,7 +656,7 @@ struct icaltimetype icaltime_null_time(void)
 
 /**     @brief Constructor.
  *
- *      Return a null date, which indicates no time has been set.
+ *      Returns a null date, which indicates no time has been set.
```

Comment:
doxygen pls. the rest of the statements too.

Reply:
Done.